packages/backend/src/storage.ts

```diff
@@ -187,7 +213,21 @@ export default (
     router.get('/new/:ensName/getConversations', async (req, res, next) => {
         try {
             const ensName = normalizeEnsName(req.params.ensName);
-            const conversations = await db.getConversationList(ensName);
+            const size =
+                parseInt(req.query.size as string) ||
```

Comment:
what would happen on size=0? Would be a frontend error obviously, but what does the backend do?

Reply:
it would return an empty array as requested by the client. There might be situations where the client might calculate such a query. This then is not inherently wrong in my opinion